tests/format/jsx/comments/jsfmt.spec.js

```diff
@@ -1,3 +1,10 @@
+run_spec(__dirname, ["flow", "babel", "typescript"], {
+  bracketSameLine: true,
+  errors: {
+    typescript: ["in-end-tag.js"],
+  },
+});
```

Comment:
I don't think we need new tests for jsx, replace all `jsxBracketSameLine` with new `bracketSameLine`.

We only need one simple test to make sure `jsxBracketSameLine` still work for `jsx`.